### DISCUSSION TITLE
[5 giugno 2023] Automi

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#10] [Automi]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/58ee242c-5cd8-437a-801e-471b13873882)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo le seguenti due soluzioni presenti nei miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/f4641913-292f-4b2b-9f5f-0ebf6ac54641)
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/5ba250d4-6f2b-4f9d-90ba-dc106b5e9e29)

----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

L'esempio di applicazione coincide con quello dell'esercizio #7

**Comment by daromacarlo** [comment]
Carissimi, è plausibile come risposta?

<img width="1429" height="591" alt="Immagine 2026-01-05 165231" src="https://github.com/user-attachments/assets/29bd0532-f721-4b73-b571-d81b9bc89b40" />
<img width="1430" height="711" alt="Immagine 2026-01-05 165246" src="https://github.com/user-attachments/assets/2177f1a7-cbd4-408b-8ad7-3d48e987b458" />
<img width="1430" height="547" alt="Immagine 2026-01-05 165300" src="https://github.com/user-attachments/assets/cb04e63b-e44a-480d-9f4c-72c10a43a898" />
